db/seeds.rb

```diff
@@ -1,11 +1,22 @@
 
-
 if Rails.env.development?
-  Game.find_or_create_by(name: "its a game",
-  description: "for srs. totlly a game", min_players: 1, max_players: 9001,
-   playing_time: "42 mins", complexity: "low")
+  User.find_or_create_by!(
+    id: 1, username: "hi",
```

Comment:
Do not specify the ID, PostgreSQL will automatically assign one for you. If you need the ID for other records, use the ActiveRecord object that is returned from this call:

```ruby
user = User.find_or_create_by!(username: "hi", ...)
game = Game.find_or_create_by!(user: user, name: "it's a game", ...)
```